lib/xclarity_client/mixins/node_mixin.rb

```diff
@@ -4,29 +4,35 @@ module XClarityClient
   #
   module Mixins::NodeMixin
     def discover_nodes(opts = {})
-      NodeManagement.new(@config).fetch_all(opts)
+      node_management.fetch_all(opts)
     end
 
     def fetch_nodes(uuids = nil,
                     include_attributes = nil,
                     exclude_attributes = nil)
-      NodeManagement.new(@config).get_object(
+      node_management.get_object(
         uuids,
         include_attributes,
         exclude_attributes
       )
     end
 
     def blink_loc_led(uuid = '')
```

Comment:
Good point, but I think that is better to put those operations on its correspondents mixins, i.e., node power operations goes on `node_mixin` and switch's stays on `switch_mixin`. This way I will remove `power_management_mixin`. Do you agree?

Reply:
@CharlleDaniel @douglasgabriel It depends on the way it works. I mean, if there's a "power management" endpoint or it was developed in a generic way, I mean: it is totally independent of other entities, make sense for me add it in, but if this mixin is required inside each entity which has power operations, it doesn't make sense add these methods inside `power_management_mixin.rb`.

Reply:
Agreed. Actually the `power_management_mixin` is a facade mixin, included in the main facade `client.rb`, where the power operations of nodes was living, but, once that now we have separated mixins by endpoints, and the operations really depends on these endpoint, I believe that make more sense to put the operations inside the correspondent mixin, i.e.,  put the node operations inside `node_mixin` and the switch's inside `switch_mixin`.

Reply:
@douglasgabriel @walteraa I aggreed, make more sense remove the `power_management_mixin` and move the node operations to the current mixin. Having said that I will approve the changes and merge because @douglasgabriel already moved that.